examples/no-repeated-options.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+// How might I throw if an option is repeated?
```

Comment:
```suggestion
// Q: How might I throw if an option is repeated?
```

Reply:
Thanks for comments. I don't think adding a Q is appropriate here, as not Q&A style. But it does make me wonder if using a brief question is a weak way of introducing the example. I was thinking of it as some context, or a problem, or a challenge. And perhaps too concise/lazy!

I'll try reworking into a more explicit setting for the example.